src/core/css/SearchResult.scss

```diff
@@ -83,4 +96,37 @@
   float: right;
   height: 38px;
   width: 38px;
+
+  .SearchResult--theme & {
+    display: block;
+    float: right;
+    height: 64px;
+    object-fit: cover;
+    object-position: top right;
+    width: 100%;
+
+    [dir=rtl] & {
+      object-position: top left;
```

Comment:
One thing, I'm not sure if a real RTL browser will show a theme back to front? If so we should transform the images and do the same thing in the details page too.